### DISCUSSION TITLE
Ability to watch for tags of a project

### DISCUSSION THREAD

**Discussion by Amatewasu** [community/community#36132] [Repositories]
A lot of projects don't publish proper releases but publish tags.
It would be nice to be able to watch for new tags (in the custom option) to be alerted when a new version of the package is published.

![image](https://user-images.githubusercontent.com/747613/195842489-713dca1a-3ea5-42f4-9e48-037e66f23415.png)

**Comment by somebody1234** [comment]
i love it when it's impossible to get notified for updates to a massive project like nw.js
i'd imagine linux distro package maintainers would find this more than a little disappointing...
